src/PerfView/SupportFiles/HtmlReportUsersGuide.htm

```diff
@@ -120,30 +160,56 @@ <h4>Expected Win from Background JIT Compilation</h4>
+    <hr />
+    <h3>
+        <a id="UnderstandingTieredCompilation">Understanding Tiered Compilation</a>
+    </h3>
+    <p>Tiered compilation is a feature that was introduced in .Net Core 2.1. It improves both startup performance and steady-state performance by hot-swapping between different compilations of the same method at runtime. </p>
+    <ol>
+        <li>Startup perf wins - The runtime requests that the JIT use minimal optimizations the first time a method is compiled. Later if the method is called frequently the method will be recompiled with more optimizations. This recompilation occurs on a background thread in parallel with other activity.</li>
+        <li>Steady-state perf wins - The runtime will identify frequently called methods whose code was originally loaded from ReadyToRun (aka crossgen) images and recompile it using the JIT. The jitted code is often more performant than the original because it can take advantage of additional information that is only known at runtime.</li>
+    </ol>
+    <p>Different compilations of the same method are refered to as tiers, and as of .Net Core 2.1 there are two of them, 'Tier0' and 'Tier1'. Tier0 is the initial code for each method, regardless whether it was obtained from the JIT or from a ReadyToRun image. Tier1 refers to the optimized jitted code that is compiled on a background thread</p>
+    <h4>Enabling tiered compilation</h4>
+    <p>In .Net Core 2.1 tiered compilation is an opt-in preview feature. It can be enabled either by any of these mechanisms:</p>
+    <ol>
+        <li>Set an app config switch in runtimeconfig.json "System.Runtime.TieredCompilation": "true"</li>
+        <li>Set the msbuild property &lt;TieredCompilation&gt;true&lt;/TieredCompilation&gt; in the application's project file</li>
+        <li>Set the environment variable COMPlus_TieredCompilation=1</li>
+    </ol>
+    <h4>Understanding Tiered Compilation events</h4>
+    <p>Each process shows a high level summary table indicating JIT time broken down by trigger. One of these triggers is 'Tiered Compilation Background.' This category contains all the Tier1 methods. 
+    The tier0 methods aren't identified explicitly as they can come from several sources:</p>
+    <ol>
+        <li>If the method is present in a precompiled ReadyToRun image then the JIT was not run. There is no accounting in the JITStats view for code loaded from images.</li>
+        <li>The method could be jitted by a foreground thread just prior to its first execution, in which case it is contained in the 'Foreground' group</li>
+        <li>If the Background JIT feature is enabled the method's usage may have been accurately predicted and jitted in advance on the Multicore JIT background thread, in which case it is accounted for in the 'Multicore JIT Background' group</li>
+    </ol>
+    <p>In the individual method listings the column 'Trigger' contains the value 'TC' for each Tier1 background recompilation due to Tiered compilation</p>
```

Comment:
nit ->  go ahead and capitalize Compilation (since you capitalized Tiered and you use the abreviation TC).